website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -68,10 +68,12 @@ For example, if a mandatory task fails and an advisory task succeeds, the run fa
 
 The following example shows a run that failed due to a mandatory run task.
```

Comment:
these I want to discuss. First, if we remove them, then we need to remove the sentences preceding them that are like "the following example shows" 

I think I feel good about taking them out entirely though. I don't know that they're adding much to the user's understanding here.